torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -801,6 +798,14 @@ def create_from_config(cls, config: CppMicroGemmConfig):
                 ):
                     continue
                 block_m, block_n, block_k = config.register_blocking
+                if (
+                    config.vec_isa_cls == VecAMX
+                    and m < block_m
+                    and input_dtype == torch.bfloat16
+                    and input2_dtype == torch.int8
+                ):
+                    # For int8 WoQ GEMM, AMX micro-kernel may not perform well if m < block_m
```

Comment:
In that case, is AVX512 micro-gemm performs better than ATen?

Reply:
Yes